docs(blog): add screen reader improvement post (#3079)
uber/baseweb: documentation-site/pages/blog/screen-reader-improvements/index.mdx

import Layout from '../../../components/layout';
import {Meta} from '../../../components/blog';
import metadata from './metadata.json';
import {Button} from 'baseui/button';

export default Layout;

<Meta data={metadata} />

Assistive technologies like [screen readers](https://webaccess.berkeley.edu/ask-pecan/what-is-a-screen-reader) are essential to people who are blind, vision impaired, or have learning disabilities. Providing support for them to operate on your web application or design system components is absolutely foundational to ensure equal functionality amongst all users. Over the past month the Base Web team has made it a priority to improve screen reader usability. In this blog post I will describe some of changes made, and provide suggestions for how you can do a similar audit of your application.

## Is my application accessible when using Base Web?

> If you're using Base Web rest assured that a significant portion of your application will be fine, but accessible components are not an entirely accessible application. You absolutely need to audit your application to be sure all every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.

Reviewer: ```suggestion
If you're using Base Web, rest assured that a significant portion of your application will be fine, but having accessible components does not mean you have an entirely accessible application. You absolutely need to audit your application to be sure every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot, right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.
```

> If you're using Base Web rest assured that a significant portion of your application will be fine, but accessible components are not an entirely accessible application. You absolutely need to audit your application to be sure all every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.

Reviewer 2: https://w3c.github.io/aria-practices/#Listbox -> https://w3c.github.io/aria-practices/

> If you're using Base Web, rest assured that a significant portion of your application will be fine, but having accessible components does not mean you have an entirely accessible application. You absolutely need to audit your application to be sure every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot, right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices/#Listbox) document.

Author: ```suggestion
If you're using Base Web, rest assured that a significant portion of your application will be fine, but having accessible components does not mean you have an entirely accessible application. You absolutely need to audit your application to be sure every visitor can successfully use it. Consider the amount of code in your app that does not come from Base Web; it's a lot, right? We've done our best to hone the published components, but it's up to you to become familiar with best-practices. A good resource for learning more is the [WAI-ARIA Authoring Practices](https://w3c.github.io/aria-practices) document.
```
